e2e-version/README.md

```diff
@@ -0,0 +1,135 @@
+# E2E Grafana version resolver
+
+This Action resolves what versions of Grafana to use when E2E testing a Grafana plugin in a Github Action.
+
+## Features
+
+The action supports two modes.
+
+**plugin-grafana-dependency (default)**
+The will return all the latest patch release of every minor version of Grafana since the version that was specified as grafanaDependency in the plugin.json. This requires the plugin.json file to be placed in the `<root>/src` directory.
+
+### Example
+
+At the time of writing, the most recent release of Grafana is 10.2.2. If the plugin has specified >=8.0.0 as `grafanaDependency` in the plugin.json file, the output would be:
+
+```json
+[
+  "10.2.2",
+  "10.1.5",
+  "10.0.9",
+  "9.5.14",
+  "9.4.17",
+  "9.3.16",
+  "9.2.20",
+  "9.1.8",
+  "9.0.8",
+  "8.5.27",
+  "8.4.11",
+  "8.3.11",
+  "8.2.7",
+  "8.1.8",
+  "8.0.7"
+]
+```
+
+Please note that the output changes as new versions of Grafana are being released.
+
+**version-support-policy**
+This will resolve versions according to Grafana's plugin compatibility support policy. This means it will resolve the latest patch release for every minor version for the current major version of Grafana, plus the most recent patch release for the last minor of the previous major version of Grafana.
```

Comment:
Not sure if this is any better...

```suggestion
This will resolve versions according to Grafana's plugin compatibility support policy. Specifically, it retrieves the latest patch release for each minor version within the current major version of Grafana. Additionally, it includes the most recent release for the latest minor version of the previous major Grafana version.```

Reply:
Yes much better, thanks!